frontend/src/components/PlayButton/PlayButton.js

```diff
@@ -6,8 +6,9 @@ const PlayButton = ({ playSection, isPlaying, className="" }) => {
     const [clicked, setClicked] = useState(false);
 
     return (
+        <>
         <div
-            className={classNames("aha__play-button border-outside", "btn", {
+            className={classNames("aha__play-button btn-blue border-outside", "btn", {
```

Comment:
Doesn't this mean that *all* PlayButtons will be blue from now on? Okay for now as a fix, but we should (re)introduce the option to override the play button color.